pallets/gear/src/tests.rs

```diff
@@ -801,7 +810,9 @@ fn lazy_pages() {
     });
 }
 
+// TODO: ignored until #642 implemented
```

Comment:
Do we really can't avoid it change change the demo a bit?
Or you some proxy contract?

Reply:
Programs in both tests are sending message with value to a user:
`(import "env" "gr_send_wgas" (func $send (param i32 i32 i32 i64 i32 i32) (result i32)))`

and as I suppose it is a required part of test logic.